File: server/types/aboutTypes.go
package types

type Employee struct {
	ID           string `json:"id"`
	Name         string `json:"name"`
	Email        string `json:"email"`
	Role         string `json:"role"`
	ProfileImage string `json:"profile_image"`
	Created      string `json:"created"`
}

type Timeline struct {
	ID      string `json:"id"`
	Title   string `json:"title"`
	Date    string `json:"date"`
	Body    string `json:"body"`
	Created string `json:"created"`
}

File: client/src/types/aboutTypes.ts
export interface Employee {
    id?: string;
    name: string;
    email: string;
    role: string;
    profile_image: string;
    created?: string;
}

export interface Timeline {
    id?: string;
    title: string;
    date: string;
    body: string;
    created?: string;
}

export interface Privacy {
    id?: string;
    title: string;
    body: string;
    created?: string;
}

export interface Terms {
    id?: string;
    title: string;
    body: string;
    created?: string;
}
